### ISSUE TITLE
Asthmatic NPC sneezes and stops crafting "Rick finished crafting."

### ISSUE THREAD

**Issue by Brambor** [CleverRaven/Cataclysm-DDA#74464]
### Describe the bug

I tell NPC to craft something. They do it for a while. They then have an asthma attack or something. They stop crafting and don't resume it afterwards.

### Attach save file

N/A

### Steps to reproduce

Probably:

1. Make NPC.
2. Let it start crafting.
3. Give it asthma attack by debug menu.
4. They stop crafting and log says they finished the task.

### Expected behavior

They resume crafting. It is annoying when they don't.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.4412 (22H2)
- Game Version: 0.G-10098-g9e287b8451 [64-bit] - Master was 5d0da40857
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by osuphobia** [comment]
#73820

**Comment by Brambor** [comment]
Yup, maybe add "paused for a moment" for a minute and then return to crafting.

One minute is 60*100 = 6 000 turns, so it would run 6 000 times less than what #73820 had to solve. Which is low enough. Maybe even just 5 secs wait would be enough.
